runtime/setup.py

```diff
@@ -360,7 +361,7 @@ def generate_version_py():
   custom_package_suffix = ""
 
 setup(
-    name=f"iree-runtime{PACKAGE_SUFFIX}{custom_package_suffix}",
+    name=f"iree-runtime{custom_package_suffix}{PACKAGE_SUFFIX}",
```

Comment:
The other way round we get packages like "iree_runtime_gcmn_testing_instrumented" whereas I think we want "iree_runtime_instrumented_gcmn_testing" as we refer to "iree-runtime-instrumented" as a package